cypress/support/elements/codap-table-elements.js

```diff
@@ -0,0 +1,8 @@
+const CodapTableElements = {
+    DEFAULT_SAGE_MODELER_DATA_TABLE: '#codap div.dg-container-view.ace div.dg-case-table-component-view:nth-child(3)',
+    EXPERIMENTS_TABLE_DEFAULT_SAGE_MODELER_DATA: 'div.dg-component-container div.dg-hier-table-view div.sc-split-view div.dg-case-table-view:nth-child(3)',
+    STEPS_TABLE_DEFAULT_SAGE_MODELER_DATA: 'div.dg-component-container div.dg-hier-table-view div.sc-split-view div.dg-case-table-view:nth-child(7)',
```

Comment:
Is there a better way of finding these divs without using nth-child? Can you find it using the table title "SageModeler Data", and collection title "Experiments" and "Data points". this may make it less brittle.

Reply:
We can try to get by the table title, but we need to navigate the dom using parent function to get to the right elements.